float64[6] joints
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose
